mne/io/edf/edf.py

```diff
@@ -20,6 +20,49 @@
 from ..constants import FIFF
 from ...filter import resample
 from ...externals.six.moves import zip
+from ...utils import copy_function_doc_to_method_doc
+
+
+def get_edf_events(raw):
+    """Get original EDF events as read from the header.
+
+        For GDF, the values are returned in form
```

Comment:
Did you mean to indent this so it would render as a block quote?

https://1661-35097627-gh.circle-artifacts.com/0/tmp/circle-artifacts.TMai80t/html/generated/mne.io.get_edf_events.html#mne.io.get_edf_events